the-example-app/README.md

```diff
@@ -0,0 +1,22 @@
+What is `The Example App model`?
+================================
+
+The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
+Contentful. If you want to see the model in action, we do have a hosted version here: https://contentful-example-app.herokuapp.com/
+
+Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space. Doing so will be
```

Comment:
I think removing the entire "Doing so..." statement actually makes it more readable:

Through our [command line app](https://github.com/contentful/contentful-cli) you can push this model to your own space:

```
 contentful space seed the-example-app
 ```